packages/eslint-config-typescript/index.js

```diff
@@ -16,6 +16,7 @@
 module.exports = {
   parser: '@typescript-eslint/parser',
   parserOptions: {
+    project: ['tsconfig.json', 'packages/web-scripts/tsconfig.json'],
```

Comment:
This change seems to be causing `yarn lint` to fail with the following error since the file doesn't exist:

```
0:0  error  Parsing error: Cannot read file '/users/yuriteixeira/projects/spotify/web-scripts/tsconfig.json'
```

```sh
❯ ll /users/yuriteixeira/projects/spotify/web-scripts/tsconfig.json
ls: /users/yuriteixeira/projects/spotify/web-scripts/tsconfig.json: No such file or directory
```

Reply:
It might be unrelated (you've added `packages/web-scripts/tsconfig.json`, and `yarn lint` is complaining about lack of `tsconfig.json` in the project's root dir.

Reply:
The eslint upgrade requires `parserOptions.project` to be set, but this is definitely not the right config

Reply:
paired and removed